firedrake/preconditioners/fdm.py

```diff
@@ -197,8 +200,9 @@ def allocate_matrix(self, V, J, bcs, fcp, pmat_type, use_static_condensation):
         :arg fcp: form compiler parameters to assemble coefficients
         :arg pmat_type: the `PETSc.Mat.Type` for the blocks in the diagonal
         :arg use_static_condensation: are we assembling the statically-condensed Schur complement on facets?
+        :arg use_amat: are we computing the Schur complement exactly?
 
-        :returns: 2-tuple with the preconditioner :class:`PETSc.Mat` and a list of assembly callables
+        :returns: 3-tuple with the stiffness and preconditioner :class:`PETSc.Mats` and a list of assembly callables
```

Comment:
I expect that :class:`PETSc.Mats` won't resolve correctly in the RST, please can you check?